### PR TITLE
fix: LEAP-420: Update README.md and update env files

### PR DESCRIPTION
**Pull request by juliosgarbi**
Update the README.md from all apps and libs and configure css-prefix per app.

### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [x] Frontend
- [x] Docs

### Does this PR introduce a breaking change?
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
- [ ] e2e
- [ ] integration
- [ ] unit

### CODE REVIEW

File: web/libs/editor/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+- **`yarn lsf:watch`: Build LSF continuously**
+  - Crucial for development, this script continuously builds Label Studio Frontend (LSF), allowing developers to observe their changes in real-time within the Label Studio environment.
+- **`yarn lsf:serve`: Run LSF standalone**
+  - To run Label Studio Frontend in standalone mode. Visit http://localhost:3000 to use the application in standalone mode.
+- **`yarn lsf:e2e`: Execute end-to-end (e2e) tests on LSF**
+  - To run comprehensive e2e tests, ensuring the frontend works as expected from start to finish. The Label Studio environment must be running, typically at `http://localhost:8080`.
+- **`yarn lsf:integration`: Run integration tests**
+  - To conduct integration tests using Cypress, verifying that different parts of LSF work together correctly.
+- **`yarn lsf:integration`: Run integration tests in UI mode**
```
Comment (bmartel) [inline web/libs/editor/README.md]: This command looks the same as above, is there a `yarn lsf:integration:ui`?

File: web/README.md [hunk 2]
```diff
@@ -2,50 +2,36 @@
 
 Label Studio is a complex, NX-managed project divided into three main components:
 
-## Main App (`apps/labelstudio`)
+## [Main App (`apps/labelstudio`)][lso]
```
Comment (bmartel) [inline web/README.md:5]: can we link from here to the README of the app/libraries?

**Review by bmartel — APPROVED** [review]

File: web/README.md (hunk 2 above)
Comment (hlomzik) [reply web/README.md:5]: as I understand it leads there: https://github.com/HumanSignal/label-studio/pull/5081/files#diff-4a534cf0dab53faad7a19d1926cb05f63ebf793e827bdf11a4a19e0e654ef639R67
```
[lso]: apps/labelstudio/README.md
```

File: web/libs/editor/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- **Multi-Format Support:** Annotate diverse datasets, including images, text, audio, and video files.
+- **Interactive Annotation Tools:** Utilize tools like bounding boxes, polygons, keypoints, and brush tools for images; text highlight tools for text; and time-series labeling for audio/video.
+- **Real-Time Collaboration:** Enables simultaneous work on projects by multiple annotators.
+- **Data Filtering and Sorting:** Sort and filter data for prioritized tasks or specific data reviews.
+- **Integration with Machine Learning Models:** Connect with ML models for pre-labeling and active learning.
+- **Extensibility and Customization:** Enhance functionality with custom scripts and integrate with external tools.
+
+### Usage Instructions
+
+DataManager provides specific scripts for operation and testing:
```
Comment (hlomzik) [inline web/libs/editor/README.md]: typo, it should be `LSF`?

File: web/libs/editor/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+- **Customizable Labeling Interfaces:** Create tailored interfaces for different data types like text, images, audio, and video using configurable templates.
+- **Multi-Format Support:** Annotate diverse datasets, including images, text, audio, and video files.
+- **Interactive Annotation Tools:** Utilize tools like bounding boxes, polygons, keypoints, and brush tools for images; text highlight tools for text; and time-series labeling for audio/video.
+- **Real-Time Collaboration:** Enables simultaneous work on projects by multiple annotators.
+- **Data Filtering and Sorting:** Sort and filter data for prioritized tasks or specific data reviews.
+- **Integration with Machine Learning Models:** Connect with ML models for pre-labeling and active learning.
+- **Extensibility and Customization:** Enhance functionality with custom scripts and integrate with external tools.
+
+### Usage Instructions
```
Comment (hlomzik) [inline web/libs/editor/README.md:5]: Where should we run these instructions from? this dir or the main one I guess?

File: web/libs/editor/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Label Studio Frontend is packed with features designed to enhance the annotation process:
+
+- **Customizable Labeling Interfaces:** Create tailored interfaces for different data types like text, images, audio, and video using configurable templates.
+- **Multi-Format Support:** Annotate diverse datasets, including images, text, audio, and video files.
+- **Interactive Annotation Tools:** Utilize tools like bounding boxes, polygons, keypoints, and brush tools for images; text highlight tools for text; and time-series labeling for audio/video.
+- **Real-Time Collaboration:** Enables simultaneous work on projects by multiple annotators.
+- **Data Filtering and Sorting:** Sort and filter data for prioritized tasks or specific data reviews.
+- **Integration with Machine Learning Models:** Connect with ML models for pre-labeling and active learning.
+- **Extensibility and Customization:** Enhance functionality with custom scripts and integrate with external tools.
```
Comment (hlomzik) [inline web/libs/editor/README.md]: those are not for LSF. and the latest one I'm very curious about — do we really have this? %)

**Review by hlomzik — CHANGES_REQUESTED** [review]
some small comments, overall looks great!

File: web/libs/editor/README.md (hunk 4 above)
Comment (juliosgarbi) [reply web/libs/editor/README.md:5]: It doesn't matter, you must run it inside web folder or its sub folders, but I can detail it!

File: web/libs/editor/README.md (hunk 5 above)
Comment (juliosgarbi) [reply web/libs/editor/README.md]: Yeah, maybe I've mixed features from LSF and Label Studio in general, I think we don't need a feature section here to be honest, I'm going to remove it!

File: web/libs/editor/README.md (hunk 4 above)
Comment (hlomzik) [reply web/libs/editor/README.md:5]: wow, never thought it's possible! than I like it a lot!

**Review by hlomzik — APPROVED** [review]

**Review by hlomzik — APPROVED** [review]

**Pull request merged**